### DISCUSSION TITLE
AdGuardHome/PiHole setup

### DISCUSSION THREAD

**Discussion by sergiobaiao** [unifi-utilities/unifi-common#224] [Q&A]
Hi there,

I've been struggling to setup AdGuardHome on my UDMP, specially on the network setup, which i didn't understood. I have my network set as 192.168.0.0/16. 

To setup AdGuardHome, the readme says i need to create a new corporate network and give it a VLAN. To do that, i need to create a new network with different ip addresses.

So, here my pain starts:

configuration variables:
VLAN=5
IPV4_IP="10.254.254.1"
IPV4_GW="10.254.254.1/24"

I've created a new corporate network, gave it the ip address of 10.254.254.1/24 and fixed the 10-dns.sh with this. I've also picked my br5.mac MAC address and put on 20-dns.conflist
{
  "cniVersion": "0.4.0",
  "name": "dns",
  "plugins": [
    {
      "type": "macvlan",
      "mode": "bridge",
      "master": "br5",
      "mac": "1E:17:03:D0:E0:67",
      "ipam": {
        "type": "static",
        "addresses": [
          {
            "address": "10.254.254.1/24",
            "gateway": "10.254.254.1"
          }
        ],
        "routes": [
          {"dst": "0.0.0.0/0"}
        ]
      }
    }
  ]
}

But after i ran 10-dns.sh and created the container, the internet stopped working because my 192.168.0.0/16 machines were unable to resolve dns requests. I've double checked and the adguard container was running, so i don't know what i've done wrong here. 

Can you help?

**Comment by justfly1111** [comment]
This isn't an answer but am pushing this up because at the exact moment I a. Having the same exact type of issue
